fe/fe-core/src/test/java/com/starrocks/analysis/SelectStmtTest.java

```diff
@@ -271,4 +271,14 @@ public void testBanSubqueryAppearsInLeftSideChildOfInPredicates()
             Assert.assertTrue(e.getMessage().contains("Subquery in left-side child of in-predicate is not supported"));
         }
     }
+
+    @Test
+    public void testGroupByAllConstants() throws Exception {
+        FeConstants.runningUnitTest = true;
+        starRocksAssert.getCtx().getSessionVariable().setOptimizerExecuteTimeout(1000000000);
+        String sql = "select cast(k1 as int), count(distinct [skew] cast(k2 as int)) from db1.tbl1 group by cast(k1 as int)";
```

Comment:
data skew usually happens on some columns instead of the query,  so I think the hint should be placed on column,  Expr and ScalarOperator should be the right place.

Reply:
Join skew and sort skew seems can not be resolved by adding random column,  any better suggestion?